src/features/user_profile/components/SettingsForm/ReferenceAreaInfo/ReferenceAreaInfo.tsx

```
@@ -36,18 +36,20 @@ export function ReferenceAreaInfo() {
           </div>
         </>
       ) : (
         <>
-          <Text type="long-m" className={s.hint}>
+          <Text type="long-m">
             {i18n.t('profile.reference_area.select_are_on_the_map')}
           </Text>
-          <div className={s.clickableText} onClick={() => goTo('/map')}>
-            {i18n.t('profile.reference_area.set_the_reference_area')}
+          <div>
+            <a className={s.link} onClick={() => goTo('/map')}>
```

Biome (1.9.4)

[error] Use a button element instead of an a element.

Anchor elements should only be used for default sections or page navigation
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)